include/sound/sof/debug.h

```diff
@@ -0,0 +1,36 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
```

Comment:
commit message:

This file content describes firmware memory allocation retrieved at run-time, typically to detect memory leaks.

Reply:
Updated